.github/workflows/custom_check.yml

```diff
@@ -12,10 +12,10 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Set up Python 3.7
+    - name: Set up Python 3.10
       uses: actions/setup-python@v4
       with:
-        python-version: 3.7
+        python-version: '3.10'
```

Comment:
Note I had to add quotes around the version here, because the yaml interpreter thought it was a numeric literal and was making it 3.1.....and Python 3.1 isn't available anymore!